gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -150,7 +150,7 @@ def bread(start_sector, number_sectors)
       encoded_data = @parser.output_to_attribute(run_correct_powershell(read_script))
       t2           = Time.now.getlocal
       buffer       = ""
-      Base64.decode64(encoded_data).split(' ').each { |c| buffer += c.to_i.chr }
+      Base64.decode64(encoded_data).split(' ').each { |c| buffer << c.to_i.chr }
```

Comment:
@jerryk55 Ouch, that sounds incredibly wasteful and inefficient, why do they do that? There's no way of just encoding the block without adding spaces (like every other encoding scheme I've ever used)?

Another possible issue. It seems you're splitting on space after the decode. What if there are spaces in the decoded string?